src/components/LoginButton.js

```diff
@@ -0,0 +1,13 @@
+import { Button } from '@material-ui/core';
+import React from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { login, logout } from '../lib/auth-state-slice';
+
+export default function LoginButton() {
+  const currentAuthState = useSelector((state) => state.authState.value);
```

Comment:
Let's use `ramda` to compose our selector functions. So you can replace `(state) => state.authState.value` with `path(['authState', 'value'])`.